deploy/kubernetes/helm/che/templates/resource-monitor-role.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+{{- if not (contains "<" .Values.global.cheWorkspacesNamespace) }}
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: resource-monitor
```

Comment:
```suggestion
  name: che-resource-monitor
```

Reply:
Referring to `workspace-view-role.yaml` and `exec-role.yaml`, their name don't have `che-` prefix.
It's the reason why I didn't add `che-` prefix.

I'm not particular about the prefix.
But I suppose it's better to add the `che-` prefix to all `*-role.yaml` for consistency.